### PR TITLE
Voting app example

### PR DESCRIPTION
**Pull request by bquenin**
This is the voting app example from docker (https://github.com/docker/example-voting-app) adapted to AMP.

## To test
```
amp cluster create
amp --server=localhost stack deploy -c examples/stacks/voting-app/voting.stack.yml voting
```

Then you should be able to reach:
 - the voting app:  https://vote.local.atomiq.io
 - the result app: https://result.local.atomiq.io

### CODE REVIEW

**Review by neha-viswanathan — APPROVED** [review]

File: examples/stacks/voting-app/LICENSE [hunk 1]
```diff
@@ -1,191 +0,0 @@
-
```
Comment (subfuzion) [inline examples/stacks/voting-app/LICENSE]: We shouldn't remove Docker's license.

**Review by subfuzion — CHANGES_REQUESTED** [review]
I don't we should remove the files that are associated with the project (we can append our own Copyright statement after Docker's in the LICENSE), although I think it's ok to rename the original README.md to Instavote.md and provide a link to it from our README.md -- which is missing!

File: examples/stacks/voting-app/LICENSE (hunk 1 above)
Comment (bquenin) [reply examples/stacks/voting-app/LICENSE]: I'll add it back

**Review by subfuzion — APPROVED** [review]

**Review by subfuzion — APPROVED** [review]
Nice! 👍

**Pull request merged**